addons/sale_crm/models/crm_lead.py

```diff
@@ -82,6 +82,7 @@ def action_new_quotation(self):
             'default_source_id': self.source_id.id,
             'default_company_id': self.company_id.id or self.env.company.id,
             'default_tag_ids': self.tag_ids.ids,
+            'default_user_id': self.user_id.id or self.env.uid,
```

Comment:
self.env.user is already the default. I would not force any other default if no user_id is set on lead (and by the way would do the same for team_id). Something like
```
if self.team_id:
    defaults['team_id'] = self.team_id.id
if self.user_id:
    defaults['user_id'] = self.user_id.id
```